tracer/src/Datadog.Trace/ClrProfiler/AutoInstrumentation/AspNet/AspNetWebApi2Integration.cs

```diff
@@ -146,10 +146,13 @@ internal static void UpdateSpan(IHttpControllerContext controllerContext, Span s
                     tags,
                     headerTags);
 
-                tags.AspNetAction = action;
-                tags.AspNetController = controller;
-                tags.AspNetArea = area;
-                tags.AspNetRoute = route;
+                if (tags is not null)
```

Comment:
[nit] Minor and could clearly be done in a different PR if we agree on this, but if we test for nullity, I'd make the argument nullable

Reply:
This is a good idea. Unfortunately, we haven't enabled nullable reference types in these projects so we can't act on this just yet 😞

Reply:
to the backlog!